src-plugins/manualRegistration/manualRegistration.h

```diff
@@ -92,6 +91,10 @@ class MANUALREGISTRATIONPLUGIN_EXPORT manualRegistration : public itkProcessRegi
     */
     virtual bool writeTransform(const QString& file);
 
+    void setParameter(int data, int channel);
```

Comment:
(that's because some of them do not work, as we saw a long time ago. I'll have to test again)